client/bundles/Main/components/App.jsx

```diff
@@ -24,6 +25,7 @@ export class App extends React.Component {
 				<Route exact path ='/' render={() => <Main sessions={this.state.sessions} />} />;
 				<Route exact path ='/register' render={() => <Registration />} />;
 				<Route exact path ='/login' render={() => <Login />} />;
+				<Route exact path ='/addsession' render={() => <AddSession />} />;
```

Comment:
Just my opinion: instead of url - sessions/addsession, we can use url - sessions/add

Reply:
We will keep addasession to stick to the convention we all ready have on the naming.